docs/src/regridder.md

```diff
@@ -15,8 +15,8 @@ ClimaCoupler.Regridder.write_to_hdf5
 ClimaCoupler.Regridder.read_from_hdf5
 ClimaCoupler.Regridder.dummmy_remap!
 ClimaCoupler.Regridder.remap_field_cgll_to_rll
-ClimaCoupler.Regridder.land_sea_mask
-ClimaCoupler.Regridder.update_masks!
+ClimaCoupler.Regridder.land_fraction
```

Comment:
I know it might become longer and you might have favored brevity, but wouldn't it be more consistent and clear to have `land_surface_fraction` and `ocean_surface_fraction` throughout? Just an idea

Reply:
I think it's quite clear since the land and ocean are just surface models in this Coupler version, but happy to change it if it is unclear to a fresh eye. @juliasloan25, shall we put this to a vote? We can also call these `area_fractions` if we want to be more explicit.

Reply:
Yeah, to me `land_surface_fraction` or `land_area_fraction` and similar for ocean would sound more clear. But up to you guys!

Reply:
I think `land_fraction` and `ocean_fraction` make sense since the coupler is using surface models, as Lenka said.